ReactiveObjCBridge.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = "2.0"
 
   s.source       = { :git => "https://github.com/ReactiveCocoa/ReactiveObjCBridge.git", :tag => "#{s.version}" }
-  s.source_files = "ReactiveObjCBridge/*.{swift}"
+  s.source_files = "ReactiveObjCBridge/*.{swift,h,m}"
```

Comment:
You probably need to specify the module map here, and the SwiftSupport.h should be excluded from `source_files` & included as `private_headers` instead.